src/components/Bar.jsx

```diff
@@ -23,6 +24,11 @@ class Bar extends Component {
     this.toggleClaudy = this.toggleClaudy.bind(this)
   }
 
+  async componentWillMount () {
+    const claudyActions = await this.store.getClaudyActions()
+    this.setState({ appsList, claudyActions })
+  }
```

Comment:
I am asking myself if `componentWillMount` is a good place to make asynchronous calls.
In [the documentation](https://facebook.github.io/react/docs/react-component.html#componentwillmount) there is nothing about it. But I see something in previous documentation referenced [in this blog](https://devbutze.com/p/131).